### ISSUE TITLE
Implement Bitrate Information

### ISSUE THREAD

**Issue by ussu99** [fethica/FRadioPlayer#3]
Is it possible to Show the Bitrate of the current Stream?

**Comment by fethica** [comment]
@ussu99 It's not supported yet, interesting feature it could be implemented in the future!

**Comment by Said13** [comment]
@fethica please implement bitrate information

**Comment by raskri** [comment]
@fethica yes this would be a great supplement to this fantastic player, please implement. :)
